### DISCUSSION TITLE
Remove creator in template selection dropdown

### DISCUSSION THREAD

**Discussion by iexos** [outline/outline#5975] [Ideas]
Currently, when selecting a template from dropdown during document creation, the second line mentions the user that initially created the document. This doesn't feel right to me: How does this help me in selecting the correct template? Documents feel like they don't belong to an individual, and I feel like templates shouldn't either. Also, its not possible to change the "owner", only be deleting an recreating it. I suggest removing this line or mentioning the collection it resides in instead.

**Comment by tommoor** [comment]
Agreed, converted to issue
